recipe/meta.yaml

```diff
@@ -66,7 +66,7 @@ requirements:
     - cudnn                                  # [cuda_compiler_version != "None"]
     - numpy
     - pytorch
-    - pytorch =*={{ torch_proc_type }}*
+    - pytorch =*=*{{ torch_proc_type }}*
```

Comment:
@Tobias-Fischer, we might actually want to revert this change as it would allow users to install `pytorch` from the `pytorch` channel as mentioned at https://github.com/conda-forge/mmcv-recipe-feedstock/pull/31#issuecomment-2096974819. I think @yit-b was trying to pull in a newer version of Pytorch according to https://github.com/conda-forge/pytorch3d-feedstock/issues/10, and wanted to relax this constraint, but we should actually be pushing for conda-forge to have a newer version of Pytorch 2.2 rather than allowing channel mixing.

Reply:
Hmm, good point, thanks for making me aware of this, I should have looked more carefully. Are you happy to open a PR?

Reply:
Yep, opened at https://github.com/conda-forge/pytorch3d-feedstock/pull/11.